src/argparse.lua

```diff
@@ -1204,7 +1314,21 @@ function ElementState:invoke()
    return self.open
 end
 
+function ElementState:check_choices(argument)
+   if self.element._choices then
+      for _, choice in ipairs(self.element._choices) do
+         if argument == choice then
+            return
+         end
+      end
+      local choices_list = "'" .. table.concat(self.element._choices, "', '") .. "'"
```

Comment:
Do we need to consider escaping choices here for the error message?

Reply:
What would it need to be escaped for? It's just getting printed

Reply:
You can do some pretty funky things by tricking a program into emitting the right terminal escape sequences.
But I don't think we really need to mitigate that here: choices are program provided, not end-user provided.